pkg/tmpl/sanitizer.go

```diff
@@ -183,18 +205,18 @@ func normalizeURLs(s string, settings *print.Settings) string {
 	return s
 }
 
-// generateIndentation generates indentation of Markdown headers
-// with base level of provided 'settings.MarkdownIndent' plus any
+// generateIndentation generates indentation of Markdown and AsciiDoc headers
+// with base level of provided 'settings.IndentLevel' plus any
 // extra level needed for subsection (e.g. 'Required Inputs' which
 // is a subsection of 'Inputs' section)
-func generateIndentation(extra int, settings *print.Settings) string {
-	var base = settings.MarkdownIndent
+func generateIndentation(extra int, settings *print.Settings, char string) string {
```

Comment:
Can you make sure `settings` is the last arguments?

Reply:
Done 👍